tests/lora/conftest.py

```diff
@@ -80,13 +84,15 @@ def dist_init():
 def dist_init_torch_only():
     if torch.distributed.is_initialized():
         return
+    backend = "nccl"
+    if current_platform.is_cpu():
+        backend = "gloo"
+
     temp_file = tempfile.mkstemp()[1]
-    torch.distributed.init_process_group(
-        backend="nccl",
-        world_size=1,
-        rank=0,
-        init_method=f"file://{temp_file}",
-    )
+    torch.distributed.init_process_group(world_size=1,
+                                         rank=0,
+                                         init_method=f"file://{temp_file}",
+                                         backend=backend)
```

Comment:
@DarkLight1337  Are our CPU vendor willing to provide hardware testing for these? Who should we contact?

Reply:
I remember there is no numa node restriction for CPU TP, so I think we can just change `VLLM_CPU_OMP_THREADS_BIND=48-92` to something like `VLLM_CPU_OMP_THREADS_BIND=48-70|71-92` to enable TP, even if these cores are on same numa node.

Reply:
My bad, I meant to ask if the CPU LoRA testing should be placed [here](https://github.com/vllm-project/vllm/blob/main/.buildkite/run-cpu-test.sh). If so, we might need to contact the CPU vendor?

Reply:
My bad, I meant to ask if the CPU LoRA testing should be placed [here](https://github.com/vllm-project/vllm/blob/main/.buildkite/run-cpu-test.sh). If so, we might need to contact the CPU vendor?